tests/linux-sandbox-cert-test.nix

```diff
@@ -0,0 +1,29 @@
+{ fixed-output }:
```

Comment:
This test is not called anywhere.

(However VM tests are pretty expensive, so maybe it's overkill to have a separate test for this...)